src/components/dataDisplays/DataDisplay.jsx

```diff
@@ -49,6 +49,8 @@ export default function DataDisplay({
   initiallySelectedRow = null,
   onSelectRow = Function.prototype,
   hideFilterSearch = false,
+  hideFilterColumns = false,
+  hideDownloadCsv = false,
```

Comment:
Thanks for this. I noted the need for this a few weeks back and totally forgot about it.